app/controllers/public_revisions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class PublicRevisionsController < ApplicationController
+
+  def new
+    @public_revision = PublicRevision.new
+    @public_revision.submission = Submission.find(params[:submission_id])
```

Comment:
Wiem, że to jest źle, zasugerowałam się wstępnie kontrolerem do ArticleRevision. Nie wiem jak przypisać dane submission do zgłoszenia jeśli nie jest to opcja wychodząca od danego zgłoszenia, czy nie powinnam jednak poprosić w formularzu użytkownika o podanie tytułu albo id?